src/cares_wrap.cc

```diff
@@ -1740,14 +1740,11 @@ void SetServers(const FunctionCallbackInfo<Value>& args) {
   uint32_t len = arr->Length();
 
   if (len == 0) {
-    int rv = ares_set_servers(channel->cares_channel(), nullptr);
+    int rv = ares_set_servers_csv(channel->cares_channel(), nullptr);
     return args.GetReturnValue().Set(rv);
   }
 
-  std::vector<ares_addr_port_node> servers(len);
-  ares_addr_port_node* last = nullptr;
-
-  int err;
+  std::string servers;
```

Comment:
Doesn't make a difference unless we run a sanitizer, i guess:

```suggestion
  std::string servers{};
```